### PR TITLE
add old RCL logger details

### PR DESCRIPTION
**Pull request by rumachan**
This is a long one retrospectively adding details of the old logger and sensors in Ruapehu Crater Lake in a way that will make them compatible with LRDCP when the data are transferred over from FITS to Tilde.

### CODE REVIEW

**Comment by elidana** [comment]
hi @rumachan , some really impressive work you did here, thank you so much!! 
i'm not sure my review will capture all those tricky bits, but I tried! 

Adding some comments here , but I'm sure this is stuff you covered and I just got a bit lost.

### temperature data pre 2009
in FITS, RU001 and RU001B temperature data starts in 1955 (I assume as manually collected data). The continuous dataset starts in 2009, and this fits nicely with this pull request. I see there are some continuous data around 1995. Will those be treated as a previous `EN` install at RU001 in a future pr? 

image
image


#### lake height data pre 2009
similar question: FITS has some lake level measurements between 1970 and 1995, but continuous data are available between 2009 and 2011, and the second sensor (former RU001A) starts in 2016

image
image

File: install/streams.csv [hunk 1]
```diff
@@ -2067,8 +2067,16 @@ RTZ,10,H,H,100,false,false,false,2011-08-04T00:00:10Z,9999-01-01T00:00:00Z
 RTZ,20,B,N,50,false,false,false,2011-05-05T00:05:00Z,2020-06-16T00:00:00Z
 RTZ,20,H,N,200,false,false,true,2011-05-05T00:05:00Z,2020-08-13T01:19:00Z
 RTZ,20,H,N,200,false,false,false,2020-08-13T01:26:00Z,9999-01-01T00:00:00Z
-RU001,01,,,-600,false,false,false,2018-05-03T03:50:00Z,9999-01-01T00:00:00Z
-RU001,02,,,-600,false,false,false,2019-03-20T23:00:00Z,9999-01-01T00:00:00Z
+RU001,01,,,-3600,false,false,false,2009-04-15T02:00:00Z,2010-04-09T00:00:00Z
+RU001,01,,,-900,false,false,false,2010-04-13T12:15:00Z,2010-05-15T04:45:00Z
+RU001,01,,,-900,false,false,false,2011-04-12T21:00:00Z,9999-01-01T00:00:00Z
+RU001,01,,,-450,false,false,false,2010-05-15T04:52:30Z,2011-04-12T20:52:30Z
```
Comment (elidana) [inline install/streams.csv:2073]: minor thing, but shall all these RU001,01 entries be in chronological order? (is it passing the test, so ok if not changed)
```
elidana@hutl24201:~/tmp/delta-2/install$ grep RU001,01 streams.csv | sort -t, -nk 10
RU001,01,,,-3600,false,false,false,2009-04-15T02:00:00Z,2010-04-09T00:00:00Z
RU001,01,,,-900,false,false,false,2010-04-13T12:15:00Z,2010-05-15T04:45:00Z
RU001,01,,,-450,false,false,false,2010-05-15T04:52:30Z,2011-04-12T20:52:30Z
RU001,01,,,-900,false,false,false,2011-04-12T21:00:00Z,9999-01-01T00:00:00Z
```

File: install/streams.csv [hunk 2]
```diff
@@ -2067,8 +2067,16 @@ RTZ,10,H,H,100,false,false,false,2011-08-04T00:00:10Z,9999-01-01T00:00:00Z
...
+RU001,02,,,-3600,false,false,false,2009-04-15T02:00:00Z,2010-04-09T00:00:00Z
+RU001,02,,,-900,false,false,false,2010-04-13T12:15:00Z,2010-05-15T04:45:00Z
+RU001,02,,,-900,false,false,false,2011-04-12T21:00:00Z,2011-06-20T23:00:00Z
+RU001,02,,,-450,false,false,false,2010-05-15T04:52:30Z,2011-04-12T20:52:30Z
```
Comment (elidana) [inline install/streams.csv:2077]: similar comment as above, e.g.
```
RU001,02,,,-3600,false,false,false,2009-04-15T02:00:00Z,2010-04-09T00:00:00Z
RU001,02,,,-900,false,false,false,2010-04-13T12:15:00Z,2010-05-15T04:45:00Z
RU001,02,,,-450,false,false,false,2010-05-15T04:52:30Z,2011-04-12T20:52:30Z
RU001,02,,,-900,false,false,false,2011-04-12T21:00:00Z,2011-06-20T23:00:00Z
```

**Review by elidana — CHANGES_REQUESTED** [review]
Hi @rumachan , wow that is indeed quite a tricky one, thanks a lot for sorting that out!!

I tried to review also considering the internal documentation you prepared.

If I got things right, I'm proposing some minor changes but other than that looks pretty good for what I can tell!


#### 1) completely remove entries related to "Ruapehu Crater Lake Outlet"?

shall those entries in `install/dataloggers.csv`  also be removed??
```
dataloggers.csv:Campbell Scientific,CR6,04108,Ruapehu Crater Lake Outlet,,2022-01-17T22:40:00Z,9999-01-01T00:00:00Z
dataloggers.csv:Campbell Scientific,CR6,04115,Ruapehu Crater Lake Outlet,,2018-05-03T00:00:00Z,2022-01-17T22:39:59Z
```
one of those two dataloggers is not used elsewhere, so not sure what to do about that/where the info comes from (perhaps the 1995 measurements??)
```
elidana@hutl24201:~/tmp/delta-2$ grep --directories=skip "Campbell Scientific,CR6,04108" */*
assets/dataloggers.csv:Campbell Scientific,CR6,04108,,
install/dataloggers.csv:Campbell Scientific,CR6,04108,Ruapehu Crater Lake Outlet,,2022-01-17T22:40:00Z,9999-01-01T00:00:00Z
install/firmware.csv:Campbell Scientific,CR6,04108,CR6.Std.11.01,1901-01-01T00:00:00Z,9999-01-01T00:00:00Z,
```

if CR6,04108 not used , then entry might also need to be removed from the firwmare.csv
```
firmware.csv:Campbell Scientific,CR6,04108,CR6.Std.11.01,1901-01-01T00:00:00Z,9999-01-01T00:00:00Z,
```

#### 2) re-arrange streams.csv to improve "human readability"
see inline comment for that , not a biggie if not addressed

**Comment by rumachan** [comment]
### Thanks for noticing the 1993 to 1995 data 

I had totally forgotten (its 30 years ago!) and my checking graphs didn't go that far back as those instruments are not listed in RIM documentation (https://gns-science.atlassian.net/wiki/spaces/volcano/pages/7191790239/RCL+Ruapehu+Crater+Lake+-+Datalogger+Installation+Argos). But they are in documentation I made when I first added data to FITS (https://gns-science.atlassian.net/wiki/spaces/FTS/pages/2796421673/Ruapehu)
According that, and the publication it references (https://www.sciencedirect.com/science/article/pii/0377027395000607?ref=cra_js_challenge&fr=RR-1), the data collection site was near the outlet and is essentially the same as the observations covered in this work, as far as is known. 

The type of temperature sensor used is known, but no model or serial number, etc. **I will add the sensor type to Delta.**
The publication refers to a data logger and a programmable calculator,  but I don't understand the relationship between the two from the text. I do not consider that sufficiently accurate and understood information to try to get it in Delta. **I will add an "unknown" entry for the datalogger.**

The time range is clear from the data, 1993-12-11T00:00:00Z to 1995-04-25T19:55:12Z. According to my early FITS notes, the system was destroyed by an eruption on 29 June 1995 (VAB RUA-1995/09). **I will modify the station start time to reflect data from 1993 and the site data to show 1993 to 1995, followed by nothing until 2009.**

Data gaps resulted from the temperature not being able to record temperatures above 47 deg C. This will not be reflected in Delta.

**Comment by rumachan** [comment]
**install/dataloggers.csv**
These entries were from the LRDCP pilot project that did not provide any useful data we want to integrate into Tilde, so they should be removed. Well spotted.
```
dataloggers.csv:Campbell Scientific,CR6,04108,Ruapehu Crater Lake Outlet,,2022-01-17T22:40:00Z,9999-01-01T00:00:00Z
dataloggers.csv:Campbell Scientific,CR6,04115,Ruapehu Crater Lake Outlet,,2018-05-03T00:00:00Z,2022-01-17T22:39:59Z
```

File: install/streams.csv (hunk 1 above)
Comment (rumachan) [reply install/streams.csv:2073]: The order is required for the file to pass the tests https://github.com/GeoNet/tickets/issues/16263

File: install/streams.csv (hunk 2 above)
Comment (rumachan) [reply install/streams.csv:2077]: The order is required for the file to pass the tests https://github.com/GeoNet/tickets/issues/16263

**Comment by elidana** [comment]
thanks @rumachan 

I'll trust your judgement on not including ~1995 data, sounds a reasonable option to me given the lack of information (though if data are usable and informative we can also cover the sensor/datalogger metadata uncertainty in some way and still provide a mean to discover/use the data).

But if you're not planning to upload those to Tilde and won't be archived as "GeoNet data" then I agree that we should not add the record here. 

Once you have completed the pr with the necessary change (as a minimum remove the outlet entries) please ping me again and I can re-review! Thanks!

File: install/streams.csv (hunk 2 above)
Comment (elidana) [reply install/streams.csv:2077]: aah ok, interesting, thanks for pointing to the related issue! ignore my suggestion then

**Comment by rumachan** [comment]
@elidana No, 1993-95 data are in, only can't reasonably do datalogger and sensor information, but this should give us access to the data. Can you please re-review. I think I have this how it needs to be.

Have put more complete documentation in the ticket, rather than in this PR as it keeps everything in one place.

**Comment by rumachan** [comment]
@elidana this is now available for re-review. Thanks

**Review by elidana — APPROVED** [review]
@rumachan thanks a lot for addressing my comments!!
This looks good, and is passing the current tests.

@ozym can you please have a quick look at the solution adopted for 1993-1995 environmental sensors? (we don't know their model, but we have data that are going to be uploaded to Tilde)

**Pull request merged**